nncf/experimental/openvino_native/graph/metatypes/openvino_metatypes.py

```diff
@@ -588,8 +594,18 @@ class OVClampMetatype(OVOpMetatype):
 # Contains the operation metatypes for which bias can be applied.
 OPERATIONS_WITH_BIAS_METATYPES = [OVConvolutionMetatype,
                                   OVConvolutionBackpropDataMetatype,
+                                  #TODO: add all metatypes with bias
                                   OVMatMulMetatype]
 
+METATYPE_TO_CHANNEL_AXIS = {
```

Comment:
Could you specify `output_channel_axis` from the `METATYPE_TO_CHANNEL_AXIS` to the metatype class?

Reply:
Done